wiki/Storyboard_Scripting/en.md

```diff
@@ -2,11 +2,11 @@
 
 ![An example of scripting in .osb](img/SBS_Base.jpg "An example of scripting in .osb")
 
-In addition to the [built-in storyboard editor](/wiki/Beatmap_Editor/Design), a plain text editor, such as Notepad or [Notepad++](http://www.notepad-plus-plus.org/), can also be used to create [storyboards](/wiki/Storyboards) for beatmaps. This workflow is called **storyboarding by scripting** (*SBS*) and it allows one to be more precise (to the pixel and the millisecond) as to when sprites appear, how they behave, and so on.
+In addition to the [built-in storyboard editor](/wiki/Beatmap_Editor/Design), a plain text editor, such as Notepad or [Notepad++](https://www.notepad-plus-plus.org/), or go beyond, the [Visual Studio Code](https://code.visualstudio.com/) with osu! syntax extension, can also be used to create [storyboards](/wiki/Storyboards) for beatmaps. This workflow is called **storyboarding by scripting** (*SBS*) and it allows one to be more precise (to the pixel and the millisecond) as to when sprites appear, how they behave, and so on.
```

Comment:
saying "the Visual Studio Code with osu! syntax extension" is meaningless as long as you don't include a link to it, because many editors allow for code highlight customization, and the way you do it only lists one more editor

Reply:
```suggestion
In addition to the [built-in storyboard editor](/wiki/Beatmap_Editor/Design), a plain text editor, such as Notepad, [Notepad++](https://www.notepad-plus-plus.org/), or [Visual Studio Code](https://code.visualstudio.com/), can also be used to create [storyboards](/wiki/Storyboards) for beatmaps. This workflow is called **storyboarding by scripting** (*SBS*) and it allows one to be more precise (to the pixel and the millisecond) as to when sprites appear, how they behave, and so on.
```
it's ok now?

Reply:
yeah